queries.sql

```diff
@@ -23,3 +23,60 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- Update the 'species' column to 'unspecified' and then rollback
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals WHERE species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Great work on your query to update the `species` column in the `animal` table! 👏 However, as a best practice, it's crucial to verify the success of your changes after each transaction. To ensure that the rollback operation effectively undoes the changes, I recommend querying the `animal` table following the rollback. This step adds an extra layer of confidence in the integrity of your database.

  - By executing a simple query to retrieve the data from the `animal` table after the rollback, you can confirm whether the changes were properly rolled back and that the data remains consistent.

Happy coding! 😊🚀🍀